hw/top_earlgrey/ip_autogen/rv_plic/fpv/tb/waivers.tcl

```diff
@@ -0,0 +1,34 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# This assertion states that if the input enable (en_i) is not high and onehot input (oh_i) is true
```

Comment:
Nit: I'd probably say "nonzero" instead of "true" because it makes it more obvious that this isn't a 1-bit value.